standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -3587,6 +3590,81 @@ public ShowCompactResponse showCompact(ShowCompactRequest rqst) throws MetaExcep
     }
   }
 
+  @RetrySemantics.ReadOnly
+  public GetLatestCompactionInfoResponse getLatestCompactionInfo(GetLatestCompactionInfoRequest rqst)
+      throws MetaException {
+    GetLatestCompactionInfoResponse response = new GetLatestCompactionInfoResponse();
+    response.setDbname(rqst.getDbname());
```

Comment:
What happens if one or more parameters are `null`, or contain invalid partitions/tablenames/dbnames?

Reply:
Well, `dbname` and `tablename` seems unnecessary in `GetLatestCompactionInfoResponse` so I removed it and also added null checking for returning quickly.